### ISSUE TITLE
IE11 CORS errors in console

### ISSUE THREAD

**Issue by dandv** [rollbar/rollbar.js#109]
I'm loading Rollbar in the `<head>` at https://idorecall.com, using the standard code snippet

``` html
<script>
    var _rollbarConfig = {
      accessToken: "xxx",
      captureUncaught: true,
      payload: {
        environment: "production"
      }
    };
!function(r){function t(o){if(e[o])return e[o].exports;var n=e[o]=...
```

IE11 displays these errors in the console for each Rollbar call:

```
SEC7118: XMLHttpRequest for https://api.rollbar.com/api/1/item/ required Cross Origin Resource Sharing (CORS).
SEC7119: XMLHttpRequest for https://api.rollbar.com/api/1/item/ required CORS preflight.
```

**Comment by Crisfole** [comment]
Just tested idorecall.com (the landing page, at least) from IE11 and I was successfully able to send an `info` note to your project (item 40).

Any chance you can send me a little more info to support@rollbar.com so we can figure out what's going on for you?

**Comment by brendancf** [comment]
I see these as well on our site.  Was there any resolution to this?

**Comment by lehresman** [comment]
I am able to reproduce this very easily.

1. Run a local web server (I just reused a Rails project I had and ran "rails serve")
2. Stick a simple `rollbar.html` file in the project root with the Rollbar snippet
3. Add the following at the bottom of the `rollbar.html` file just created

```javascript
  setTimeout(function() {
    Rollbar.log('test');
  }, 2000);
```

4. Load this in IE11

Notice the console output gives CORS errors:
![screen shot 2017-02-13 at 2 11 14 pm](https://cloud.githubusercontent.com/assets/96005/22898705/4db55554-f1f6-11e6-9369-4f9f635e4ed5.png)

Tested using Rollbar JS notifier v1.9.3

**Comment by lehresman** [comment]
Looks like this is only a problem on IE11 running on Windows 7 and 8.  The error has been removed on IE11 for Windows 10.

https://msdn.microsoft.com/query/dev12.query?appId=Dev12IDEF1&l=EN-US&k=k(VS.WebClient.Help.SEC7119)

"Note  This error code was removed in IE11 on Windows 10. It remains in IE11 for Windows 8.1 and Windows 7."

**Comment by lehresman** [comment]
More research further convinces me that this is just a warning message and can be ignored.  See the discussion on http://stackoverflow.com/questions/24646732/sec7118-xmlhttprequest-cors-ie-console-message.

I have confirmed that this does not impact performance, and the Rollbar payload is still effectively delivered.  This is just a case of IE11 being excessively verbose.

Closing, since everything still works and there is nothing that can be done to silence IE11 in this case.
